.travis.yml

```diff
@@ -76,8 +76,7 @@ before_script:
   - composer install --prefer-dist --no-interaction
 
 script:
-  - if [[ $DEFAULT = 1 && $TRAVIS_PHP_VERSION = 7.* ]]; then export CODECOVERAGE=1 ; phpdbg -qrr vendor/bin/phpunit --coverage-clover=clover.xml; fi
```

Comment:
I have reported the problem to [codecov](https://github.com/codecov/support/issues/217) today and they are usually pretty quick to respond/resolve issues. So I would suggesting keep this to help debug the problem.